code/modules/fishing/fish/chasm_detritus.dm

```diff
@@ -94,14 +97,23 @@
 
 	return chasm_contents
 
+/// Body detritus is selected in favor of bodies belonging to sentient mobs
+/// The first sentient body found in the list of contents is returned, otherwise 
+/// if none are sentient choose randomly.
+/obj/item/chasm_detritus/restricted/bodies/determine_detritus(list/chasm_stuff)
+	for(var/thing in chasm_stuff)	
+		var/mob/fallen_mob = thing
+		if(fallen_mob.mind)
```

Comment:
Is there any benefit here to using `important_recursive_contents[RECURSIVE_CONTENTS_CLIENT_MOBS]` instead?
I'll be honest I'm not totally sure what the _intended_ use case for that is, but it's been handy for fishing people out with VV and might save you some cycling until someone replies to this to say "no don't use that for this reason"